Source/Core/Common/MemArena.cpp

```diff
@@ -29,6 +29,19 @@
 #endif
 #endif
 
+#ifdef _WIN32
+typedef PVOID(WINAPI* PVirtualAlloc2)(HANDLE Process, PVOID BaseAddress, SIZE_T Size,
```

Comment:
Perhaps put these in the header file so you don't need to cast the pointer every time you make call to one of these?

Reply:
I didn't do that to avoid including the horrible `Windows.h` in the header, but ymmv, especially given that it was already there before.

Reply:
Good point, either is OK for me.